components/k8s-model-server/mxnet-serving/mms_docker_cpu/mms_app.conf

```diff
@@ -0,0 +1,47 @@
+# mxnet-model-server arguments
+--models
+resnet-18=https://s3.amazonaws.com/model-server/models/resnet-18/resnet-18.model
+
+--service
+optional
+
+--gen-api
+optional
+
+--log-file
+optional
+
+--log-rotation-time
+optional
+
+--log-level
+optional
+
+--metrics-write-to
+optional
+
+# Gunicorn arguments
+--bind
+unix:/tmp/mms_app.sock
+
+--workers
+4
+
+--worker-class
+gevent
+
+--limit-request-line
+0
+
+# Nginx configurations
+server {
```

Comment:
Is this running NGINX? Do we need it?

Reply:
this is how the amazon version of mxnet seving works.. https://github.com/awslabs/mxnet-model-server
do we want to dig too deep into the implementation?